.git-blame-ignore-revs

```diff
@@ -0,0 +1,2 @@
+# style: reformat project; adding multiple pre-commit hooks (#12)
+83ff7ef9b6a6eeb954bb0124e97c3d161e561ea5
```

Comment:
unrelated to this pr, but github has this feature, where you can add specific commits to a blacklist, such that git blame shows the prior commit. useful for formatting commits.